MyStore/src/JsonParcer.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+
+
+class JsonParcer
+{
+private:
+
+	struct InfoProduct
+	{
+		std::string m_product_name = {};
+		std::string m_product_color = {};
+	}Product;
+
+	std::string m_file_path = {};
+	std::vector<InfoProduct> m_data;
+
+public:
+
+	JsonParcer() = delete;
+	JsonParcer(std::string path) : m_file_path(path) {};
+	JsonParcer(const JsonParcer& obj) = delete;
+	JsonParcer(const JsonParcer&& obj) = delete;
+	~JsonParcer() {};

```

Comment:
You can discard this line. Default destructor will be generated anyway. It makes sense to declare empty (default) destructor only when you want to mark it as virtual.